SCF-Session-5/lcmhcf/LcmHcfTest.java

```diff
@@ -0,0 +1,36 @@
+package lcmhcf;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class LcmHcfTest 
+{
+	LcmHcf lcmHcfObject = new LcmHcf();
+	
+	@Test
+	public void lcmTest() 
```

Comment:
Method name is `lcmTest` but it has a test for `hcf` also. Name should be clear about the method's functionality and there should be different test methods for each method of the class.

Reply:
done